fin/CMakeLists.txt

```diff
@@ -327,8 +318,9 @@ else()
     message("MIOPEN_ENABLE_FIN is SET")
 endif()
 
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
+find_package(miopen)
```

Comment:
why cmake inside MIOpen should find miopen?
is it copy paste?

Reply:
The fin directory is a submodule
https://github.com/ROCmSoftwarePlatform/fin